src/github.com/getlantern/flashlight/analytics/analytics.go

```diff
@@ -27,12 +28,15 @@ var (
 func Configure(cfg *config.Config, version string) {
 
 	if cfg.AutoReport != nil && *cfg.AutoReport {
-		analytics.Configure(TrackingId, version, cfg.Addr)
+		pubsub.Sub(pubsub.IP, func(ip string) {
+			log.Debugf("Got IP %v -- starting analytics", ip)
+			analytics.Configure(ip, TrackingId, version, cfg.Addr)
```

Comment:
I'm a little bit concerned about this snippet only because the UI could try to call the analytics service before it's up. That might have also been possible before not sure, and the consequences are probably very minimal, but just wanted to flag it.